SPECS/libnbd/libnbd.spec

```diff
@@ -122,26 +170,51 @@ Install this package if you want intelligent bash tab-completion
 for %{name}.
 
 %prep
+%if 0%{verify_tarball_signature} && ! 0%{?azl}
```

Comment:
Since we already disabled `verify_tarball_signature`, do we need ` && ! 0%{?azl}`?

Reply:
Fair, will clean

Reply:
```suggestion
%if 0%{verify_tarball_signature}
```